client/js/states/trading.js

```diff
@@ -0,0 +1,81 @@
+//let tradeState = temporary(new QuestionState("Would you like to trade one of your party members for 3 pounds of food?", acceptTradeState, declineTradeState))
+function tradeSupply(){
+		let item = randNonZeroKey(party.supplies);
+		let item1 = randNonZeroKey(party.supplies);
+		
+		while (item == item1){ // not trading the same thing 
```

Comment:
Possible infinite loop, it'd be better to just remove the first one so the second pick won't ever pick the first.